src/components/SignInButton.jsx

```diff
@@ -15,16 +18,25 @@ export const SignInButton = () => {
             instance.loginPopup(loginRequest).catch(e => {
                 console.log(e);
             });
+// </ms_docref_constant_sign_in_button>
```

Comment:
We really need to make sure each code block is complete on its own, including closing braces. It might require some refactoring of the sample and/or docs if there's too much overlap of redirect and popup here.

Reply:
We need to refactor the sample code, we have too much overlap